components/camel-zipkin/src/main/java/org/apache/camel/zipkin/ZipkinTracer.java

```diff
@@ -583,9 +588,12 @@ private void clientRequest(Tracing brave, String serviceName, ExchangeSendingEve
             TraceContext context = span.context();
             String traceId = "" + context.traceIdString();
             String spanId = "" + context.spanId();
+            MDC.put("traceId", traceId);
```

Comment:
Beware that some of the code you added with MDC.put are only executed if you have DEBUG logging, so you need to move this outside, or fix it in another way. This happens in a few places in this source file.

Reply:
@davsclaus , Thanks for the review comments, will do changes